workflows/mark00.smk

```diff
@@ -0,0 +1,75 @@
+"""Experiment mark00 - small test run of MCMC"""
+
+from pyggdrasil.tree_inference import McmcConfig, TreeId, TreeType, CellSimulationId, CellAttachmentStrategy
+
+
+###############################################
+## Experiment mark00
+experiment = "mark00"
+# Mutation Data / Cell Simulation
+CS_seed = 42
+n_cells = 1000
+CS_fpr = 0.4
+CS_fnr = 0.4
+CS_na = 0.0
+observe_homozygous = False
+cell_attachment_strategy = CellAttachmentStrategy.UNIFORM_EXCLUDE_ROOT
+# mutations
+mutations = 7
+nodes= mutations + 1
+# True Tree
+true_tree_type = TreeType("r")
+true_tree_seed = 5
+true_tree_n_nodes = nodes
+# Initial Tree
+initial_tree_type = TreeType("d")
+initial_tree_seed = 4545
+initial_tree_n_nodes = nodes
+# MCMC Parameters
+mcmc_seed = 42
+mcmc_config_id = McmcConfig(n_samples=100, fpr=0.4, fnr=0.4).id()
+###############################################
```

Comment:
One place to specify one MCMC chain run. - Will think of the best way to run multiple chains for the experiments.